services/src/main/java/org/keycloak/keys/AbstractEcKeyProviderFactory.java

```diff
@@ -36,15 +36,16 @@ public final static ProviderConfigurationBuilder configurationBuilder() {
         return ProviderConfigurationBuilder.create()
                 .property(Attributes.PRIORITY_PROPERTY)
                 .property(Attributes.ENABLED_PROPERTY)
-                .property(Attributes.ACTIVE_PROPERTY);
+                .property(Attributes.EC_GENERATE_CERTIFICATE_PROPERTY);
```

Comment:
I notice there is no frontend test for this switch. For instance in `cypress/e2e/realm_settings_events_test.spec.ts`:
```javascript
realmSettingsPage.toggleSwitch("ecGenerateCertificate", false);
```

I also notice there are no frontend changes to display the certificates as in `realm-settings/keys/KeysListTab.tsx` for `EC` keys the logic is hardcoded not to display the `Certificate` button. This should probably be changed to display based on the presence of the `certificate` attribute.